include/gui_card_span.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef GUI_CARD_SPAN_HPP_
+#define GUI_CARD_SPAN_HPP_
+
+#include <list>
+#include <optional>
+#include "gui_card.hpp"
+#include "raylib-cpp.hpp"
+
+namespace meow {
+class GuiCardSpan {
+private:
+    int m_card_gap = 50;
+    raylib::Window *m_window = nullptr;
+    std::list<GuiCard> m_cards;
+    std::list<std::pair<GuiCard, double>> m_removed_cards;
```

Comment:
please either write a comment which describes what does double mean in std::pair or define struct which will contain these 2 fields (I personally would prefer second option)